src/utils/asar.ts

```diff
@@ -0,0 +1,111 @@
+// @ts-ignore
+import asar from 'asar';
+import { toReadableStream } from './buffer';
+import path from 'path';
+import fs from 'fs';
+/*eslint-disable no-unused-vars*/
```

Comment:
Not nice, but out of this PR's scope. I created a new issue about it: https://github.com/DevExpress/testcafe-hammerhead/issues/2068